editor/register_editor_types.cpp

```diff
@@ -230,10 +231,16 @@ void register_editor_types() {
 
 	GLOBAL_DEF("editor/version_control/plugin_name", "");
 	GLOBAL_DEF("editor/version_control/autoload_on_startup", false);
+
+	EditorInterface::create();
 }
 
 void unregister_editor_types() {
 	EditorNode::cleanup();
+
+	if (EditorInterface::get_singleton()) {
```

Comment:
Another redundant check, I think. `free()` method already checks if singleton exists. With an error, but this method shouldn't be called before calling `register_editor_types()`, so it would be wrong anyway.